src/js/jenkins_loader.js

```diff
@@ -116,26 +116,26 @@ JenkinsLoader.prototype.loadBuilds = function (jobName, callback) {
 
 JenkinsLoader.prototype._parseBuilds = function (jobUrl, jobName, builds, callback) {
     // convert from `build -> targets` to `target -> builds` mapping
-    var targetBuilds = {};
+    const targetBuilds = {};
 
-    var targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
+    const targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
 
     builds.forEach(build => {
         build.artifacts.forEach(relativePath => {
-            var match = targetFromFilenameExpression.exec(relativePath);
+            const match = targetFromFilenameExpression.exec(relativePath);
 
             if (!match) {
                 return;
             }
 
-            var version = match[1];
-            var target = match[2];
-            var date = new Date(build.timestamp);
+            const version = match[1];
+            const target = match[2];
+            const date = new Date(build.timestamp);
 
-            var formattedDate = ("0" + date.getDate()).slice(-2) + "-" + ("0" + (date.getMonth()+1)).slice(-2) + "-" +
+            const formattedDate = ("0" + date.getDate()).slice(-2) + "-" + ("0" + (date.getMonth()+1)).slice(-2) + "-" +
                 date.getFullYear() + " " + ("0" + date.getHours()).slice(-2) + ":" + ("0" + date.getMinutes()).slice(-2);
```

Comment:
Maybe this needs a little refactor. Simple divide into several variables (day, month, year, hours, minutes) each one will contain the slice and finally use them in the formattedDate. It will improve a lot the reading of that.
Is not mandatory, the purpose of this PR is to change the lexical scope, but I think will make a better code with little effort.

Reply:
Refactored as requested and tested.